tests/test_output_formatter.cpp

```diff
@@ -318,6 +318,33 @@ auto construct_find_multiple_reply_duplicate_image()
 
     return reply;
 }
+
+auto construct_version_info_multipassd_update_available()
+{
+    auto reply = mp::VersionReply();
+    reply.set_version("Daemon version");
+    reply.set_log_line("Some logline goes here");
+
+    mp::UpdateInfo* updateInfo = new mp::UpdateInfo; // Leak? expecting protobuf to deal with memory dealloc.
+    updateInfo->set_version("update version number");
+    updateInfo->set_title("update title information");
+    updateInfo->set_description("update description information");
+    updateInfo->set_url("http://multipass.web");
+
+    reply.set_allocated_update_info(updateInfo);
+
+    return reply;
+}
+
+auto construct_version_info_multipassd_up_to_date()
+{
+    auto reply = mp::VersionReply();
+    reply.set_version("Daemon version");
+    reply.set_log_line("Some logline goes here");
```

Comment:
And this?

Reply:
Same as above, `log_line` is redundant and was created back when I did not fully understand the RPC message components.

Reply:
Same, just meant the log line.